adam/adam_6224_ds/adam_6224.py

```diff
@@ -0,0 +1,355 @@
+
+# -*- coding: utf-8 -*-
+#
+# This file is part of the dev-solaris-adam project
+#
+#
+#
+# Distributed under the terms of the LGPL license.
+# See LICENSE.txt for more info.
+
+""" Adam6224
+It is a definicion of a class used to control ADAM-6224 controller via Modbus/TCP.
```

Comment:
No, packages should be named with `snake_case`. Only in case of `CamelCase` you should keep the original capital letters of "ADAM" and we use `CamelCase` for classes and Tango interface elements (attributes, properties and commands).